src/main/java/fr/fallenvaders/minecraft/plugin/controller/PlayerControllerImpl.java

```diff
@@ -203,14 +205,28 @@ public void openEnderChest(@NotNull Player playerSender) {
   @Override
   public void openDisposal(@NotNull Player playerSender) {
     playerSender.openInventory(
-        server.createInventory(
-            playerSender,
-            27,
-            miniMessage.deserialize(
-                resourceBundle.getString("fallenvaders.essentials.disposal.inventory.title"))));
+      server.createInventory(
+        playerSender,
+        27,
+        miniMessage.deserialize(
+          resourceBundle.getString("fallenvaders.essentials.disposal.inventory.title"))));
     messageController.sendInfoMessage(playerSender, essentialsMessages.openDisposalInventory());
   }
 
+  @Override
+  public void setHat(@NotNull Player playerSender) {
+    ItemStack itemInHand = playerSender.getEquipment().getItemInMainHand();
+
+    if (itemInHand.getType().isAir()) {
+      messageController.sendErrorMessage(playerSender, essentialsMessages.noItemInHandHat());
```

Comment:
Ce n'est pas une erreur : tout fonctionne correctement. Par contre, c'est un "failure" puisque l'utilisateur n'a pas manipulé la commande dans les bonnes conditions.
=> `sendFailureMessage`

```suggestion
      messageController.sendFailureMessage(playerSender, essentialsMessages.noItemInHandHat());
```

Reply:
Le changement n'a pas été appliqué